tests/core/reward_model/test_sequence_classification.py

```diff
@@ -10,19 +10,23 @@ def reward_model() -> SequenceClassificationRewardModel:
 
 
 def test_batch_judge(reward_model: SequenceClassificationRewardModel) -> None:
-    num_instances = 4
     instances = [
         RewardBenchInstance(
             prompt=[{"role": "user", "content": "Hello!"}],
             chosen=[{"role": "assistant", "content": "Hi!"}],
             rejected=[{"role": "assistant", "content": "Hello!"}],
-            extra_info={"id": i},
-        )
-        for i in range(num_instances)
+            extra_info={"id": 1},
+        ),
+        RewardBenchInstance(
+            prompt=[{"role": "user", "content": "1 + 1 = ?"}],
+            chosen=[{"role": "assistant", "content": "2"}],
+            rejected=[{"role": "assistant", "content": "3"}],
+            extra_info={"id": 2},
+        ),
```

Comment:
Can we assume that these samples have different lengths?

Reply:
Good point, it wasn't obvious...
I've updated the example to make it clear that the second one is obviously longer than the first

Reply:
Looks great! Thanks for addressing my concerns 👍